tools/schemacode/bidsschematools/validator.py

```diff
@@ -476,206 +533,17 @@ def write_report(
-                "when calling the validator.",
-                schema_version,
-                schema_min_version,
-                schema_min_version,
-            )
-            schema_version = schema_min_version
-    schema_dir = os.path.join(schema_reference_root, schema_version)
-    if os.path.isdir(schema_dir):
-        return schema_dir
-    else:
-        raise ValueError(
-            f"The expected schema directory {schema_dir} does not exist on the system. "
-            "Please ensure the file exists or manually specify a schema version for "
-            "which the bidsschematools files are available on your system."
-        )
-
-
-def log_errors(validation_result):
-    """
-    Raise errors for validation result.
-
-    Parameters
-    ----------
-    validation_result : dict
-        A dictionary as returned by `validate_all()` with keys including "schema_tracking",
-        "path_tracking", "path_listing", and, optionally "itemwise".
-        The "itemwise" value, if present, should be a list of dictionaries, with keys including
-        "path", "regex", and "match".
-    """
-    total_file_count = len(validation_result["path_listing"])
-    validated_files_count = total_file_count - len(validation_result["path_tracking"])
-    if validated_files_count == 0:
-        lgr.error("No valid BIDS files were found.")
-    for entry in validation_result["schema_tracking"]:
-        if entry["mandatory"]:
-            lgr.error(
-                "The `%s` regex pattern file required by BIDS was not found.",
-                entry["regex"],
-            )
-    for i in validation_result["path_tracking"]:
-        lgr.warning("The `%s` file was not matched by any regex schema entry.", i)
-
-
-def _get_directory_suffixes(my_schema):
-    """Query schema for suffixes which identify directory entities.
-
-    Parameters
-    ----------
-    my_schema : dict
-        Nested directory as produced by `bidsschematools.schema.load_schema()`.
-
-    Returns
-    -------
-    list of str
-        Directory pseudofile suffixes excluding trailing slashes.
-
-    Notes
-    -----
-    * Yes this seems super-awkward to do explicitly, after all, the trailing slash is
-        already in so it should automagically work, but no:
-        - Subdirectory names need to be dynamically excluded from validation input.
-        - Backslash directory delimiters are still in use, which is regrettable.
-    """
-    pseudofile_suffixes = []
-    for i in my_schema["objects"]["extensions"].values():
-        i_value = i["value"]
-        if i_value.endswith("/") and i_value != "/":
-            pseudofile_suffixes.append(i_value[:-1])
-    return pseudofile_suffixes
-
-
 def validate_bids(
     in_paths,
-    accept_dummy_paths=False,
-    schema_reference_root="{module_path}/data/",
-    schema_version=None,
+    dummy_paths=False,
+    bids_reference_root="/usr/share/bids-schema/versions",
+    schema_path=BUNDLED_SCHEMA_PATH,
```

Comment:
Again, these should be `None`.

Reply:
fixed in [29c0a4d](https://github.com/bids-standard/bids-specification/pull/1398/commits/29c0a4d15375ac1aed6e7deeb8f1fa5973aeead5)